vizro-core/docs/pages/user-guides/filters.md

```diff
@@ -64,7 +64,7 @@ automatically based on the target column, e.g. a dropdown for categorical data,
 
 ## Changing selectors
 
-If you want to have a different selector for your filter, you can provide the `selector` argument of the [`Filter`][vizro.models.Filter] with a different selector model.
+If you want to have a different selector for your filter, you can pass the `selector` argument of the [`Filter`][vizro.models.Filter] with a different selector model.
```

Comment:
I saw this replacement a couple of times now and it always sounds a bit odd to me, but I'm no native speaker 😄 Isn't it **pass to** something instead of *pass with*. Everytime `provide` was replaced with `pass` it sounds a bit odd?

Reply:
And here I think it should be the other way around if we use "pass". You can pass a different selector model to the `selector` argument of the Filter. The current phrasing confuses me a lot